### PR TITLE
Adding Gulp to technologies list in readme file

### PR DESCRIPTION
**Pull request by vbadkar**

### CODE REVIEW

File: includes/blogdb.sql [hunk 1]
```diff
@@ -0,0 +1,139 @@
+-- phpMyAdmin SQL Dump
```
Comment (JayKandari) [inline includes/blogdb.sql:1]: This File doesn't seem to be a part of this issue. Can we remove this file from this PR. and create a separate one for including this database file?

Comment (JayKandari) [reply includes/blogdb.sql:1]: @vbadkar ^

**Review by JayKandari — CHANGES_REQUESTED** [review]
Added a comment.

File: includes/config.php [hunk 2]
```diff
@@ -1,128 +0,0 @@
-<?php
-require 'db.php';
```
Comment (JayKandari) [inline includes/config.php:2]: We don't require this file? why is this file removed?

**Review by JayKandari — CHANGES_REQUESTED** [review]

File: includes/config.php (hunk 2 above)
Comment (vbadkar) [reply includes/config.php:2]: This file was added by mistake and is not required in this pull request, hence removing it
